bug: fluent manager fails to compile with "undefined: NewOptions"

### Describe the issue

https://github.com/fluent/fluent-operator/pull/1776/files re-factored fluent-manager's code and split `NewOptions` out from `main.go` to `options.go` and now `make binary` is failing:

```
Error: cmd/fluent-manager/main.go:71:10: undefined: NewOptions
```

### To Reproduce

Run `make binary`

### Expected behavior

The binary builds.

--- a/Makefile
+++ b/Makefile
@@ -110,8 +110,8 @@ verify-codegen:
 
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/fluent-manager ./cmd/fluent-manager
-go build -o bin/fb-watcher ./cmd/fluent-watcher/fluentbit
-go build -o bin/fd-watcher ./cmd/fluent-watcher/fluentd
+	go build -o bin/fb-watcher ./cmd/fluent-watcher/fluentbit
+	go build -o bin/fd-watcher ./cmd/fluent-watcher/fluentd
 
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run cmd/fluent-manager/main.go

